docs/content/guide/installation/index.md

```diff
@@ -59,6 +59,21 @@ docker run \
     latest
 ```
 
+**CAUTION** : When you create a new change file with the `new` command, without specify required flags,
+you must add the `it` ([interactive terminal](https://docs.docker.com/engine/reference/commandline/run/#assign-name-and-allocate-pseudo-tty---name--it)) option,
+otherwise, it won't run as expected.
```

Comment:
I feel like caution may be a bit much for the message, it is not like it will delete your data or anything. Also add the details about `--user`.

```suggestion
**Notes**
1. In order to complete prompts with docker you will need to use an [interactive terminal](https://docs.docker.com/engine/reference/commandline/run/#assign-name-and-allocate-pseudo-tty---name--it)
1. You may also want to include your own user and group ID if any files would be created using the [user option](https://docs.docker.com/engine/reference/run/#user).

```sh
docker run \
    --mount type=bind,source=$PWD,target=/src \
    -w /src \
    -it \
    --user $(id -u ${USER}):$(id -g ${USER}) \
    ghcr.io/miniscruff/changie \
    new
```

```

Reply:
Agree with your suggestion, except perharps for admonition usage `**Notes**`.
[Admonitions](https://www.markdownguide.org/hacks/#admonitions) suggest singular syntax and not plural value.

Reply:
I think there examples were all singular but plural should be just fine since we do have two ( and may add more considering docker has quite a few rules ). We could separate each special case into a separate header under docker but that feels a bit much. We are mostly describing an example scenario and I would expect the end user to modify it to there own use cases.